Tests.Integration.Presentation.Web/Interfaces/V2/ItInterfaceApiV2Test.cs

```diff
@@ -452,6 +451,40 @@ public async Task GET_Many_As_Stakeholder_With_Name_Filter()
             Assert.Equal(dto.Uuid, itInterface1.Uuid);
         }
 
+        [Fact]
+        public async Task GET_Many_As_Stakeholder_With_UsedInOrganizationUuid_Filter()
+        {
+            //Arrange
+            var (token, organization) = await CreateStakeHolderUserInNewOrg();
+            var orgId = organization.Id;
+            var system = await ItSystemHelper.CreateItSystemInOrganizationAsync(A<string>(), orgId, AccessModifier.Public);
+            var system2 = await ItSystemHelper.CreateItSystemInOrganizationAsync(A<string>(), orgId, AccessModifier.Public);
+            var system3 = await ItSystemHelper.CreateItSystemInOrganizationAsync(A<string>(), TestEnvironment.DefaultOrganizationId, AccessModifier.Public);
+
+            var validDto = InterfaceHelper.CreateInterfaceDto(A<string>(), A<string>(), orgId, AccessModifier.Public);
+            var invalidNoExhibitDto = InterfaceHelper.CreateInterfaceDto(A<string>(), A<string>(), orgId, AccessModifier.Public);
+            var invalidNoUsageDto = InterfaceHelper.CreateInterfaceDto(A<string>(), A<string>(), orgId, AccessModifier.Public);
+            var invalidWrongOrganizationDto = InterfaceHelper.CreateInterfaceDto(A<string>(), A<string>(), TestEnvironment.DefaultOrganizationId, AccessModifier.Public);
+
+            var validInterface = await InterfaceHelper.CreateInterface(validDto);
+            await  InterfaceHelper.CreateInterface(invalidNoExhibitDto);
+            var invalidNoUsageInterface = await InterfaceHelper.CreateInterface(invalidNoUsageDto);
+            var invalidWrongOrganizationInterface = await InterfaceHelper.CreateInterface(invalidWrongOrganizationDto);
+
+            await ItSystemHelper.TakeIntoUseAsync(system.Id, orgId);
```

Comment:
remember to take it system3 into use in the wrong org if the idea is to validate that it is excluded because you search for the org within the usage on the system